pykeyvi/requirements.txt

```diff
@@ -0,0 +1,5 @@
+msgpack-python
+autowrap
```

Comment:
Good point.
I think it's better to ask `autowrap` guys to confirm or add an official API.